baby_bingo.py

```diff
@@ -0,0 +1,60 @@
+
+########## SUPPORTING PACKAGES ##########
+import random
+import re
+import sys
+
+
+########## SUPPORTING FUNCTIONS ##########
+
+def remove_number_prefix(my_string):
+    m = re.match('^[0-9]*[.]\s(.*)', my_string)
+    return(m.group(1))
+
+def remove_rating_suffix(my_string):
+    m = re.match('([a-zA-Z]*)\s(No change)*[+-]*[0-9]*', my_string)
+    return m.group(1)
+
+
+########## CODE TO BE EXECUTED ##########

```

Comment:
the main part of this code could be moved to a `main` function which would be called at the end of the file.
```
if __name__ == "__main__":
  main()
```